test/e2e/auth/service_accounts.go

```diff
@@ -517,6 +518,118 @@ var _ = SIGDescribe("ServiceAccounts", func() {
 			framework.Failf("Unexpected error: %v\n%s", err, logs)
 		}
 	})
+
+	// TODO(mtaufen): Mark as an alpha feature test (need to remember how to do that)
+	ginkgo.It("should support OIDC discovery of service account issuer [Feature:OIDCDiscovery]", func() {
+		// TODO(mtaufen): Remove the RBAC setup here so we are actually testing access via public-info-viewer
```

Comment:
I feel like we shouldn't add anything else to public-info-viewer.

Reply:
The KEP included adding this to public-info-viewer.

Reply:
It says "we will consider". I softened the words on that before lgtm'ing.